packages/manager/modules/bm-server-components/src/netboot/controller.js

```diff
@@ -168,17 +168,15 @@ export default class BmServerComponentsNetbootCtrl {
           this.$translate.instant('server_configuration_netboot_success'),
         );
```

Comment:
mabe should add this.loading.setNetboot = false; in the end of the block ?

Reply:
For this scenario I still need to retain loader for success case thats why I removed finally block and moved this.loading.setNetboot = false; into catch block.